src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/FilterMeta.cs

```diff
@@ -1,12 +1,53 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
 namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
 
-public class FilterMeta
+public class FilterMeta : ICreatedUpdatedTimestamps<DateTimeOffset, DateTimeOffset?>
 {
-    public required string Identifier { get; set; }
+    public int Id { get; set; }
```

Comment:
`Guid`

Reply:
**Discussed:**

Only using `Guids` for publicly available IDs. Using `int` for strictly internally available IDs as it is more performant.